bleak/backends/corebluetooth/PeripheralDelegate.py

```diff
@@ -369,6 +380,19 @@ def did_update_value_for_characteristic(
     ) -> None:
         c_handle = characteristic.handle()
 
+        if not error:
+            notification_discriminator = (
+                self._characteristic_notification_discriminators.get(c_handle)
+            )
+            if notification_discriminator and notification_discriminator(
+                bytearray(value)
```

Comment:
I would use `bytes` here. The use of `bytearray` elsewhere is something of an odd choice, but can't change it now without breaking users.

Reply:
Now using bytes.